### PR TITLE
[APEX-208] Fixed style violations in StreamingContainer.java

### PR DESCRIPTION
**Pull request by ilganeli**
Fixed code style issues in StreamingContainer.java

### CODE REVIEW

File: engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java [hunk 1]
```diff
@@ -920,8 +927,8 @@ private void deployNodes(List<OperatorDeployInfo> nodeList) throws IOException
   }
 
   private HashMap<String, ComponentContextPair<Stream, StreamContext>> deployOutputStreams(
-    List<OperatorDeployInfo> nodeList, HashMap<String, ArrayList<String>> groupedInputStreams)
-    throws Exception
+      List<OperatorDeployInfo> nodeList, HashMap<String, ArrayList<String>> groupedInputStreams)
```
Comment (tweise) [inline engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java]: Is this a checkstyle violation or an IDE settings issue?

Comment (ilganeli) [inline engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java]: @tweise @chandnisingh What would you want this to look like? Is it with regards to `throws Exception` being on the new line? This seems to be an IDE Settings issue, code style does not adjust it one way or the other. The relevant setting seems to be `throws keyword` to `wrap if long` but it makes no difference. I think from a style perspective both are valid.

Comment (otterc) [inline engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java]: Isn't the change adding more spaces? continuation indentation looks like more than 2 spaces. I am using apex style for IntelliJ and that has continuation indentation set to 2. 

The problem with apex style (for IntelliJ which is checked in) is that it aligns method parameters which I will fix.

**Comment by tweise** [comment]
Any update on this? Do the style settings need change or reformat the file?

**Comment by vrozov** [comment]
Actually we need both. We need to enforce wrapping indentation in the checkstyle (it is set to 2 and is not currently enforced) and change back wrapping indentation to level 4 in StreamingContainer.java.

**Comment by ilganeli** [comment]
@vrozov @chandnisingh I can make these updates once the appropriate code style settings are checked in.

**Comment by otterc** [comment]
@ilganeli I made the changes to strictly enforce wrapping indentation and also corrected the intellij style configuration. Max line length is not enforced yet. I will follow up on that discussion and push that change soon.

**Comment by tweise** [comment]
@ilganeli Diff looks good. Can you please rebase/squash the commits and include the the JIRA into the commit message:  http://apex.incubator.apache.org/contributing.html

Thanks!

**Comment by ilganeli** [comment]
@tweise Oh boy, what have I done. I think I messed up the rebase :-/. Working on fixing this.

**Comment by ilganeli** [comment]
Not sure why GitHub is still showing all these separate commits but the commits above should now be squashed

**Comment by tweise** [comment]
Ilya, please create a new branch from the head of devel-3 and then do

```
git merge --squash StyleFixesStreamingContainer 
```

That will leave you with a single commit only containing the changes you made. Cleanup the commit message and then push it back to this branch.

Same can be accomplished using rebase --onto also.

**Comment by ilganeli** [comment]
@tweise Trying `git push origin StyleFixesStreamingContainer` from the new branch after the merge just says "Everything up to date". Is there another step that I'm missing?

**Comment by tweise** [comment]
git push origin HEAD:StyleFixesStreamingContainer -f

Let me know how that goes.

**Comment by ilganeli** [comment]
@tweise Beautiful! Great trick, thanks!

**Comment by tweise** [comment]
@ilganeli thanks for following through and helping test the code style settings.

**Pull request merged**